### DISCUSSION TITLE
Grouping of prompts

### DISCUSSION THREAD

**Discussion by mindspawn** [open-webui/open-webui#3011] [Ideas]
Would be nice to have the ability to group / (slash) prompts e g. A group for writing, some for coding etc

**Comment by JuanjoJG-WATA** [comment]
What I do to have a similar result to this is always start the prompts with whatever they do, like: 
"**/writting-poems, /writting-short-text, /code-python, /code-javascript**"

This way I just need to write /code or /writting and then just select whatever I want. At the moment it's the closest thing you can do, I believe.

**Reply by mindspawn** [reply]
True. Same here as well :) was hoping to avoid the duplicated prefixes.
